pkg/web/apps/apps_http_handler_test.go

```diff
@@ -6,29 +6,39 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/aerogear/mobile-security-service/internal/db"
+	"github.com/aerogear/mobile-security-service/pkg/config"
+
 	"github.com/labstack/echo"
 )
 
-var mockPostgresRepository = NewPostgreSQLRepository()
-var mockService = NewService(mockPostgresRepository)
-
 func TestHTTPHandler_GetApps(t *testing.T) {
+	config := config.Get()
+	dbConn, err := db.Connect(config.DB.ConnectionString, config.DB.MaxConnections)
```

Comment:
This makes our test dependant on the db running. We need to avoid this in our unit tests. The other mobile services split it so their integration tests have a dependency on the db but not unit tests.

Reply:
@davidffrench I have added an integration tag to the handler, repository and service test files so that they will only when the integration tag is supplied during `go test`. Is this right?

Reply:
Yes, that should do it but wasn't for me locally.

Reply:
As discussed in person, integration tests should be separated out.

Reply:
@craicoverflow Apologies, I see the changes for the integration tests have split everything out. This is not exactly what I meant. There should be a single file for the integration api tests just hitting the endpoints with the database setup. Exactly like https://github.com/aerogear/aerogear-app-metrics/blob/master/pkg/test/metrics-api_test.go have it setup. This will cover end to end integration of the components working together.

The db_test.go file can stay where it was but just with the `// +build integration` comment.

The HTTP handler, service and repository files still need there own unit test files associated with them. 

Let me know if you have any questions. If you prefer, catch me tomorrow and we can collaborate.

Reply:
:+1: